README.md

```diff
@@ -1,18 +1,11 @@
-# create-svelte
+# SvelteKit on the edge
 
-Everything you need to build a Svelte project, powered by [`create-svelte`](https://github.com/sveltejs/kit/tree/master/packages/create-svelte).
+A demo [SvelteKit](https://kit.svelte.dev) app running on [Vercel Edge Functions](https://vercel.com/features/edge-functions), which run close to your users to enable dynamic server-side rendering at the speed of static content.
 
-## Creating a project
+## How to enable edge functions in your SvelteKit app deployed to Vercel
```

Comment:
```suggestion
## How to enable edge functions in your SvelteKit app deployed to Vercel

- Update [svelte.config.js](/svelte.config.js) with a supported [adapter](https://kit.svelte.dev/docs/adapters)
> While the Vercel Build Output API is in beta, you'll need to add the `ENABLE_VC_BUILD` environment variable with a value of `1` to your project on [vercel.com](https://vercel.com). 
- Use your package manager to install [`@sveltejs/adapter-vercel`](https://github.com/sveltejs/kit/tree/master/packages/adapter-vercel)
- Update [svelte.config.js](/svelte.config.js) to use `adapter-vercel` instead of the default `adapter-auto`
- Pass the `edge: true` option to the adapter
```